### PR TITLE
Fix readme commands for msbot clone

### PR DESCRIPTION
**Pull request by daveta**
Fixes casing and command issues with msbot deploy command.
- Use fully expanded parameters (ie, --folder vs -f)
- Subdirectories match case with C#
- Add "services" to "msbot clone services".
- Add Azure login commands
- Remove (temporary) broken switch --subscriptionId

### CODE REVIEW

**Comment by cleemullins** [comment]
@daveta Could you resolve the conflicts? Looks like it's just "/" vs "\" and should be easy.

File: samples/csharp_dotnetcore/51.cafe-bot/README.md [hunk 1]
```diff
@@ -1,163 +1,173 @@
...
+## Scenarios covered via other samples
+- [Proactive messages](../../../samples/csharp_dotnetcore/16.proactive-messages)
+- [Multi-lingual conversations](../../../samples/csharp_dotnetcore/17.multi-lingual-conversations)
+- [Bot authentication](../../../samples/csharp_dotnetcore/18.bot-authentication)
+
+# To try this sample
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+
```
Comment (JasonSowers) [inline samples/csharp_dotnetcore/51.cafe-bot/README.md]: extra line of whitespace

File: samples/csharp_dotnetcore/51.cafe-bot/README.md [hunk 2]
```diff
@@ -1,163 +1,173 @@
...
+    npm i -g msbot chatdown ludown luis-cli qnamaker 
+    ```
+
+## Configure required services
+1. Follow instructions [here](https://portal.azure.com) to create an Azure account. If you already have an account, sign in. Click on all services -> search for 'subscriptions' -> copy the subscription ID you would like to use from the Home > Subscriptions page.
+2. Follow instructions [here](https://www.luis.ai/home) to create a LUIS.ai account. If you already have an account, sign in. Click on your name on top right corner of the screen -> settings and grab your authoring key.
+3. To create and configure required LUIS and QnA Maker services, 
+    - In a Powershell prompt,
+```bash
+cd BotBuilder-Samples\samples\csharp_dotnetcore\51.cafe-bot
```
Comment (JasonSowers) [inline samples/csharp_dotnetcore/51.cafe-bot/README.md]: change '\\' to '/'

File: samples/csharp_dotnetcore/51.cafe-bot/README.md [hunk 3]
```diff
@@ -1,163 +1,173 @@
...
+```
+## Visual Studio
+- Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/51.cafe-bot`) and open `CafeBot.csproj` in Visual Studio.
+- Press F5.
+## Visual Studio Code
+- Open `botbuilder-samples/samples/csharp_dotnetcore/51.cafe-bot` sample folder.
+- Bring up a terminal, navigate to `botbuilder-samples/samples/csharp_dotnetcore/51.cafe-bot` folder.
+- Type 'dotnet run'.
+
+
```
Comment (JasonSowers) [inline samples/csharp_dotnetcore/51.cafe-bot/README.md]: extra whitespace line

**Pull request merged**